sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,206 @@
+---
+timer: 'H H * * *'
+parameters: []
```

Comment:
The `fork_ami` job has a lot of parameters on Jenkins. I would have expected to see them here, too.